CMakeLists.txt

```diff
@@ -0,0 +1,147 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
The repo may (possibly) contain additional dependencies besides ffmpeg in the future...

Perhaps we can rename to `ffmpeg_cbs.cmake` or something? I don't know if we should have a directory where we place cmake files.

Reply:
I'm not sure how to do this... do I just have a dummy top-level CMakeLists.txt for now? cmake won't pick it up otherwise right?

Reply:
I think you can call it directly.

`cmake <options> ffmpeg_cbs.cmake`

Reply:
If that doesn't work...

`cmake -P <other options> ffmpeg_cbs.cmake` I think will work...

Or worst case, in the workflow, rename the file back to `CMakeLists.txt` before executing `cmake`.

Reply:
how about this?

**CMakeLists.txt**
```txt
#if (FFMPEG_CBS)
  include (${CMAKE_CURRENT_SOURCE_DIR}/ffmpeg_cbs.cmake)
#endif()
```

**command**
```bash
cmake -DFFMPEG_CBS=ON ...
```